src/js/edu-benefits/utils/options-for-select.js

```diff
@@ -453,6 +453,15 @@ const schoolTypes = [
   { label: 'National test reimbursement (for example, SAT or CLEP)', value: 'testReimbursement' },
   { label: 'Licensing or certification test reimbursement (for example, MCSE, CCNA, EMT, or NCLEX)', value: 'licensingReimbursement' },
   { label: 'Apprenticeship or on-the-job training', value: 'apprenticeship' },
+  { label: 'Correspondence', value: 'correspondence' }
+];
+
+const schoolTypesWithTuitionTopUp = [
```

Comment:
I think you could build this array like this:

`const schoolTypesWithTuitionTopUp = schoolTypes.concat({ label: 'Tuition assistance top-up (Post 9/11 GI Bill and MGIB-AD only)', value: 'tuitionTopUp' });`

`concat` is sort of like `push`, except it returns a copy of the array with the new items added to the end instead of modifying it.